ci/src/artifacts/upload.sh

```diff
@@ -38,34 +28,61 @@ rclone_common_flags=(
     --s3-no-check-bucket
 )
 
-REMOTE_SUBDIR="${REMOTE_SUBDIR:?Remote subdirectory not set}"
+log() {
+    echo "$@" >&2
+}
 
-echo "uploading $f to AWS" >&2
-AWS_PROFILE=default "$RCLONE" \
-    "${rclone_common_flags[@]}" \
-    --s3-provider=AWS \
-    --s3-region=eu-central-1 \
-    --s3-env-auth \
-    copy \
-    "$f" \
-    ":s3:dfinity-download-public/ic/${VERSION}/$REMOTE_SUBDIR/"
-echo "done uploading to AWS" >&2
+rclone() {
+    if [[ $DRY_RUN == "1" ]]; then
+        log "[rclone dry run]"
```

Comment:
I've decided not to print out the full command here in case we ever pass secrets as CLI arguments